packages/ckeditor5-upload/docs/_snippets/features/base64-upload.js

```diff
@@ -18,6 +18,14 @@ ClassicEditor
 	} )
 	.then( editor => {
 		window.editor = editor;
+
+		setTimeout( () => window.attachTourBalloon( {
```

Comment:
Why do we need the `setTimeout` function here?

Reply:
Also used in block quote.

Reply:
In the past tooltip was not attached without setTimeout in several places, but after recent changes I see that now it is no longer needed. I will remove setTimeout here and check other places where I've used it.

Reply:
Done.

Reply:
@pomek I left one setTimeout in \`[collaboration.js](https://github.com/cksource/collaboration-features/pull/3733/files#diff-62ee64bd2e36abaca2779f589489558f4c4dae3ae8a37b5e645fa976ebbf3ae1) \`: [https://github.com/cksource/collaboration-features/pull/3733/files](https://github.com/cksource/collaboration-features/pull/3733/files)  
The editor here is loading with delays and without it, the tooltip is wrongly positioned. Do you think it is not a big problem to leave it like this?

Reply:
As for now, I'd leave it. In the future, we could try to improve it because the 1s delay might not be enough.